### PR TITLE
TF-1233 Fix error out of range in create avatar icon

### PR DESCRIPTION
**Pull request by hoangdat**
Error:
image

Resolved: 
image

### CODE REVIEW

File: core/lib/presentation/extensions/string_extension.dart [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
+        final regexLetter = RegExp("([A-Za-z])");
+        final listMatch = regexLetter.allMatches(trim()).toList();
+        if (listMatch.length > 1) {
+          final firstLetter = listMatch[0].group(0);
+          final secondLetter = listMatch[1].group(0);
+          return firstLetter != null && secondLetter != null
+              ? '${firstLetter.toUpperCase()}${secondLetter.toUpperCase()}'
+              : '';
+        } else {
+          final firstLetter = substring(0, 2);
```
Comment (dab246) [inline core/lib/presentation/extensions/string_extension.dart:32]: If the word has 1 character it will jump to `catch ` -> you won't have a name avatar

Comment (hoangdat) [reply core/lib/presentation/extensions/string_extension.dart]: fixed

**Review by ManhNTX — APPROVED** [review]
LGTM

**Review by dab246 — APPROVED** [review]
sound good

**Pull request merged**